src/plugins/video-creator/backend/capture.ts

```diff
@@ -57,6 +57,7 @@ async function screenshot2d(vc: VideoCreator, size: ScreenshotOpts) {
     ...size,
     showLabels: true,
     mathBounds: clampedMathBounds,
+    mode: "contain" as "contain",
```

Comment:
`as const` also works here.

Reply:
Nice! That is cleaner. I will change it.